tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -670,7 +670,7 @@ def tearDown(self):
                 client_restarts,
                 0,
                 msg=(
-                    "Client pods unexpectedly restarted"
+                    "Client container unexpectedly restarted"
```

Comment:
Let's also make this change for the server

Reply:
Sure, changed server message too.